src/test/samples/javascript/sampleClasses.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+class NonExportedClass {
+    publicMethod() { }
```

Comment:
Does it make sense to call these `publicMethod` and `privateMethod` in JavaScript?

Reply:
Copy paste mistake. Removed privateMethod